index.html

```diff
@@ -5,5 +5,15 @@
     <title>Document</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h2>Quiz App</h2>
```

Comment:
```suggestion
      <h1>Quiz App</h1>
```
You should give your homepage a `h1` for semantic reasons, since we should never skip a headline tag in the `h1` to `h6` hierarchy. You will learn about that in the active learning in the afternoon.